### PR TITLE
Governance models image

### PR DESCRIPTION
**Pull request by Ainali**
and explanation of the difference between codebase governance and governance of implementations

-----
[View rendered activities/codebase-stewardship/supporting-governance/index.md](https://github.com/publiccodenet/about/blob/governance-models-image/activities/codebase-stewardship/supporting-governance/index.md)

### CODE REVIEW

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ The development of public code requires various forms of governance. Codebase st
 
 The Foundation for Public Code *does not* directly govern codebases. Instead, we offer support and advise on codebase governance, and help *communities* execute governance.
 
+## What is codebase governance
```
Comment (bvhme) [inline activities/codebase-stewardship/supporting-governance/index.md:11]:
```suggestion
## What codebase governance is
```

File: activities/codebase-stewardship/supporting-governance/index.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ The development of public code requires various forms of governance. Codebase st
...
+
+Codebase governance is strictly limited to the codebase. The governance of the codebase should strive to be as simple, transparent and welcoming as possible. This is to help the codebase scale, and allow different types of parties; public associations, large and small municipaities, and different companies to get involved. The current draft ‘[governance.md](governance-template.md)’ file contains a proposal, including a technical steering team and product steering team.
```
Comment (bvhme) [inline activities/codebase-stewardship/supporting-governance/index.md:13]: If we call it a 'governance file' it keeps it open for people to convert it to another file type if so needed.

```suggestion
Codebase governance is strictly limited to the codebase. The governance of the codebase should strive to be as simple, transparent and welcoming as possible. This is to help the codebase scale, and allow different types of parties; public associations, large and small municipaities, and different companies to get involved. The current draft ‘[governance file](governance-template.md)’ contains a proposal, including a technical steering team and product steering team.
```

**Comment by Ainali** [comment]
Sure, but wouldn't that let the tools dictate the content? I mean, there is a semantic difference between new line and space.

**Comment by bvhme** [comment]
In Markdown the single newline does not have a semantic meaning of breaking content since a lot of people and editors decide to wrap manually by putting a newline at the end of a rule or often at 80 characters or so. A double newline signifies a content break by ending the paragraph. A breakrule is done by two spaces followed by a newline.

So the semantic meaning stays the same, because the token for the semantic meaning commonly associated with breaking content is not one but two newlines. It does however keep 1 statement per line and since Git does version control on a per-line basis it gives some extra control. It is not letting the tools dictate the content in my opinion, it is using the tools effectively.

But this is not a breaking thing, it's just a tip to make life easier.

**Comment by Ainali** [comment]
Would
it
not
be
even
easier
like
this
then?

**Comment by bvhme** [comment]
:)

That would not be easier since at that point you lose it's that context again.

This is our codebase, our source code, every sentence in here is a statement on how we operate and it's history is an integral part of the content.

If we see every sentence as a clause it starts to make sense to have the ability to change their order without destroying them, to be able to quickly identify which one was changed and which one wasn't as well as change the composition of paragraphs.

For clear diffs: If we have a typo fix on a paragraph that is on one line, the whole paragraph would be a difference, if it is on separate lines we only get that line as a difference. If the order gets rearranged we don't get a big different paragraph we need to compare ourselves to find out it's just an order change, we see the change itself.

Additionally, if we put one statement per line it makes `git blame` work better since it will couple the commits of changes to the lines even if the lines are rearranged or the paragraphs changed.

In the end, I don't think this is too important for now, but it might save us a lot of head-aches to build up professionalism around this in the future.

Further reading: [Semantic Linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/)

**Pull request merged**